x/programs/cmd/simulator/src/lib.rs

```diff
@@ -3,7 +3,8 @@
 //! Alternatively the `Plan` can be written in JSON and passed to the
 //! Simulator binary directly.
 
-use serde::{Deserialize, Serialize};
+use base64::{engine::general_purpose::STANDARD as b64, Engine};
```

Comment:
Is there any real benefit to base64 encoding everything?

Reply:
https://github.com/ava-labs/hypersdk/issues/994